pillar/params.sls

```diff
@@ -1,13 +1,23 @@
 # the CIDR for cluster IPs (internal IPs for Pods)
 cluster_cidr:     '172.16.0.0/13'
 
-# The min and max subnet, and length for node allocations.
-# The defaults allow for 510 pod IPs per host (/23) allowing
-# for a maximum of 1024 hosts.
-cluster_cidr_min: '172.16.0.0'
-cluster_cidr_max: '172.23.255.255'
+# The size of the subnet allocated to each host.
+# Defaults to 24 (i.e. /24) unless Network was configured to be smaller than a /24 in which case it is one less than the network.
+# The defaults allow for
+# * 2^(23-13)     = 10 bits for hosts = 1024 hosts
+# * 2^(32-23) - 2 = 9 bits - 2 for pods = 510 pod IPs per host,
+#   (with IPs from '172.16.0.1' to '172.16.1.254')
+# TODO: we currebtly have 19 bits for hosts|pods, so we should ask
+#       users in the UI about the "max number of hosts"
+#       showing all the powers of 2 (ie, 256, 512, 1024), and then
+#       calculate and show the max number of pods...
 cluster_cidr_len: 23
 
+# These are automatically calculated by Flannel or
+# k8s's Network Manager
+# cluster_cidr_min: '172.16.0.0'
+# cluster_cidr_max: '172.23.254.0'
```

Comment:
I suggest to set them to `None` by default instead of commented out. The effect will be the same but we are clearly stating that our default pillar values for them are to be autocalculated unless overriden.

Reply:
I would prefer not to rely on how we deal with `None`s (and they would be the only `None`s in our pillar).

Reply:
I'm a bit worried in general about commented code because you never know if it has a purpose or it was a leftover or what... that's why I think commented out code is in general not a good practice. With our latest way to access the pillar with `salt.caasp_pillar.get` a `None` should not do any harm here...

Reply:
I think it is more confusing to leave this kind of behavior to some obscure mechanism of _how-we-deal-with-Nones_ than a comment. I will clearly state this is not a spurious comment...